main.qml

```diff
@@ -794,4 +794,8 @@ ApplicationWindow {
             }
         }
     }
+    onClosing: {
+        walletManager.closeWallet(currentWallet);
+        console.log("onClosing called");
+    }
```

Comment:
@mbg033 yes. that makes sense.
Also this one could probably be solved cleaner. 
https://github.com/monero-project/monero-core/blob/master/main.qml#L147
The previous strict comparison evaluates true when a wallet is opened that has previously crashed or has been killed by the user. Application crashes after password dialog when closeWallet is being called on that wallet.